README.md

```diff
@@ -43,14 +43,24 @@ $ pytest todo_app\tests\
 
 #### Integration testing
 
-You will need Firefox version 60+ to run these tests. [Download here](https://www.mozilla.org/en-GB/firefox/new/).
+You will need Firefox version 60+ to run visual web tests. [Download here](https://www.mozilla.org/en-GB/firefox/new/). 
 
-To run integration tests use the command:
+To run Firefox tests use the command:
 
 ```bash
-$ pytest todo_app\tests_e2e\
+$ pytest todo_app\tests_e2e\firefox_test.py
 ```
 
+Tests are also available in Chrome. You will need Chrome version 88. [Download here](https://chromedriver.storage.googleapis.com/index.html?path=88.0.4324.96/).
```

Comment:
That's a link to _Chromedriver_, not Chrome.

You should gitignore chromedriver.exe and remove it from the repo. The correct chromedriver depends both on your local Chrome version and your OS.

Similarly, looks like geckodriver is in the repo but should not be.